examples/pony/dumb_passthrough/README.md

```diff
@@ -0,0 +1,113 @@
+# Alphabet
```

Comment:
This README is for the alphabet app

Reply:
I'm removing all passthrough apps in the next commit, after the current round of CI tests finish.